vm/hv1/hvdef/src/lib.rs

```diff
@@ -1883,7 +1883,17 @@ macro_rules! registers {
                 Sifp = 0x000A0012,
                 Sipp = 0x000A0013,
                 Eom = 0x000A0014,
-                Sirbp = 0x000A0015,
```

Comment:
I removed this register because not only do we never use it, but neither does the hcl.

Reply:
`hvdef` is for _definitions_ and should not reflect what we actually support in any crate that consumes it.